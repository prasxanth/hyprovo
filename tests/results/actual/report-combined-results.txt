2021-11-24 17:25:51.777 - INFO - Pass ✓...  add pass
2021-11-24 17:25:51.781 - ERROR - Fail ✗...  sub fail
2021-11-24 17:25:51.784 - INFO - Pass ✓...  mul pass
2021-11-24 17:25:51.791 - INFO - Pass ✓...  div pass
2021-11-24 17:25:51.794 - ERROR - Fail ✗...  add fail
2021-11-24 17:25:51.797 - ERROR - Fail ✗...  mul fail
2021-11-24 17:25:51.797 - DEBUG -  Test Summary → Total: 6    Pass: 3    Fail: 3
