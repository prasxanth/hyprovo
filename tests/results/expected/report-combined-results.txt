2021-11-24 17:24:55.362 - INFO - Pass ✓...  add pass
2021-11-24 17:24:55.365 - ERROR - Fail ✗...  sub fail
2021-11-24 17:24:55.371 - INFO - Pass ✓...  mul pass
2021-11-24 17:24:55.380 - INFO - Pass ✓...  div pass
2021-11-24 17:24:55.384 - ERROR - Fail ✗...  add fail
2021-11-24 17:24:55.387 - ERROR - Fail ✗...  mul fail
2021-11-24 17:24:55.387 - DEBUG -  Test Summary → Total: 6    Pass: 3    Fail: 3
